test/agentchat/contrib/vectordb/test_pgvectordb.py

```diff
@@ -0,0 +1,123 @@
+import os
+import sys
+
+import pytest
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+
+try:
+    import pgvector
+    from autogen.agentchat.contrib.vectordb.pgvector import PGVector
+    import sentence_transformers
+except ImportError:
+    skip = True
+else:
+    skip = False
+
+
+@pytest.mark.skipif(skip, reason="dependency is not installed OR requested to skip")
```

Comment:
To run this test, some change is needed in contrib-tests.yml. You can check other retrieve chat examples.

Reply:
I'm a little unclear on how to create a test for pgvector. There is an external dependency on a postgres database. The connection details are currently hardcoded in the test_pgvectordb.py right now. Should the contrib-tests.yml also setup a postgres DB with pgvector and use the hardcoded connection string?

Reply:
Sounds OK to me, as long as it's restricted to some env only.

Reply:
@sonichi  I believe I have set this up in the contrib-tests.yml. I also added a test_pgvector_retrievechat.py to accompany that github workflow job. Please let me know if anything else is needed or if corrections are needed.